pyccel/errors/messages.py

```diff
@@ -162,3 +162,12 @@
 WRONG_LINSPACE_ENDPOINT = 'endpoint argument must be boolean'
 NON_LITERAL_KEEP_DIMS = 'keep_dims argument must be a literal, otherwise rank is unknown'
 NON_LITERAL_AXIS = 'axis argument must be a literal, otherwise pyccel cannot determine which dimension to operate on'
+MISSING_KERNEL_CONFIGURATION = 'Kernel launch configuration not specified'
+INVALID_KERNEL_LAUNCH_CONFIG_HEIGHT = "'int' object has no attribute 'handle'"
```

Comment:
Why use `HEIGHT` here? did you mean `HIGH`?

Reply:
Why would it be high? I'm not sure what is meant by "invalid kernel launch config height", but "height" is a noun so grammatically it makes sense. "invalid kernel launch config high" wouldn't